### PR TITLE
Add enterprise url to offers emails

### PR DESCRIPTION
**Pull request by raq929**

### CODE REVIEW

File: ecommerce_worker/sailthru/v1/tasks.py [hunk 1]
```diff
@@ -361,7 +361,8 @@ def send_course_refund_email(self, email, refund_id, amount, course_name, order_
 
 
 @shared_task(bind=True, ignore_result=True)
-def send_offer_assignment_email(self, user_email, offer_assignment_id, subject, email_body, site_code=None):
```
Comment (binodpant) [inline ecommerce_worker/sailthru/v1/tasks.py:364]: where is the url actually consumed? does it end up in the email sent out? sorry I have zero knowledge of this area

**Comment by binodpant** [comment]
How did we test this locally? is it possible to run that celery task locally? looks safe to me since you are adding default empty value .. not sure who / how this task is called

**Review by binodpant — APPROVED** [review]
ok on the change itself, asked a few clarifying questions on slack but no flags with this change

File: ecommerce_worker/sailthru/v1/tasks.py (hunk 1 above)
Comment (raq929) [reply ecommerce_worker/sailthru/v1/tasks.py:364]: It's consumed by Sailthru, which is our marketing email service. There's some templating code there that will handle both the case of empty string and the actual url should it get it.

**Comment by binodpant** [comment]
did recommend looking at https://github.com/edx/ecommerce-worker/blob/3e720f3fae62518f98a24e1f7421f6e64db3f165/ecommerce_worker/sailthru/v1/tests/test_tasks.py#L862 in case it needs an update but due to default value it seems not too critical

**Comment by raq929** [comment]
I did modify and test the log output for this task. 
I did not run it locally, unfortunately; I'm not sure how to do that, and this repo is proving really difficult to work with. Hopefully once it's updated to 3.5 it will be a bit more straightforward?

**Pull request merged**